pkg/apis/scheduling/types.go

```diff
@@ -183,6 +183,11 @@ type PodGroupSpec struct {
 	// if there's not enough resources to start all tasks, the scheduler
 	// will not start anyone.
 	MinResources *v1.ResourceList
+
+	// MinQuotas defines the minimal resource quota of members/tasks to run the pod group;
+	// if there's not enough resources to start all tasks, the scheduler
+	// will not start anyone.
+	MinQuotas *v1.ResourceList
```

Comment:
What's the difference between `MinResource` and `MinQuotas`? Is there a design doc for the feature?

Reply:
https://github.com/volcano-sh/volcano/pull/1583

Reply:
BTW, can we just use `MinResource` when considering resourceQuota? In other words, do we need to add a new `MinQuotas` field?

Reply:
minResouce only contains request resources， but resourceQuota contains request & limits

Reply:
> minResouce only contains request resources， but resourceQuota contains request & limits

So if `resourceQuota` contains the function of `minResource`?

Reply:
here is an example：
```
  minQuotas:
      count/pods: "1"
      cpu: "8"
      limits.cpu: "32"
      limits.ephemeral-storage: 50Gi
      limits.memory: 64Gi
      memory: 16Gi
      pods: "1"
      requests.cpu: "8"
      requests.memory: 16Gi
    minResources:
      cpu: "8"
      ephemeral-storage: 50Gi
      memory: 16Gi
```
so minQuotas contains the data of minResources。 but these two fields are for different scenarios。 and we cannot get rid of minresources.

Reply:
got it

Reply:
sorry, I'm confused about this field. IMO, we should reuse minResource.

Reply:
I think the main point here is that minResources is a value calculated based on limit and requests, but minQuotas can contain both limit and requests

Reply:
> we should reuse minResource.

as @shinytang6 said, if we reuse minResource, the meaning of this field is changed. i don't think this is what we expected

Reply:
we need pay more attentation on similar new field :)

Reply:
is there anyway to enhance minResource?

Reply:
actually minQuotas contains he data of  minResources. if we want to reduce similar fields:
1. may be we need a new field (we can named it minRequest) which calculated like minQuotas,  and then deprecated minResource field.
2. the overcommit, proportion plugin need to modify to use this new field.

Reply:
> may be we need a new field (we can named it minRequest) which calculated like minQuotas, and then deprecated minResource field.

So the differentiation is name? Why rename it?

Reply:
the field meaning and he value of this filed are both changed

Reply:
let's sync up this in weekly meeting :)

Reply:
> 

Conclusion: reserve all fields from `minQuotas` to `minResource` @merryzhou